docs/local_install_skaffold.md

```diff
@@ -1,51 +1,266 @@
->
-> Do not forget to unzip the files after download.
 
-### 2. Install Docker and Kubernetes
+### Configuration
+
+#### Minikube (Ubuntu only)
 
-First of all, download the *Docker desktop* installer from https://www.docker.com/products/docker-desktop. You'll have to create an account there to do so. Then run it to install Docker on your machine.
+> Note: If you are using Mac, this part will be handled by Docker Desktop for you; you can directly head to the Helm section. Still, you can use Minikube on Mac and select it in your Docker Desktop application.
 
-Once installed, launch Docker and open its "preferences" panel. In the Kubernetes tab, check the `Enable Kubernetes` checkbox.
+Please enable the ingress minikube module: `minikube addons enables ingress`. You might need to edit `skaffold.yaml` files and set `nginx-ingress.enabled` to `false`.
 
-Kubernetes will take a while to launch the first time, but once it is done, you can move on to configuring it as follow:
+You can now start Minikube with:
 
-### 3. Configure Kubernetes
+```sh
+cd hlf-k8s
+# Comfortable setup
+minikube start --cpus 6 --memory 8192 --disk-size 50g --kubernetes-version='v1.15.4'
+# Frugal setup
+minikube start --cpus 4 --memory 8192 --disk-size 30g --kubernetes-version='v1.15.4'
+# VM setup (Inside a VM, you will have to execute all commands with sudo)
+sudo minikube start --vm-driver=none --kubernetes-version='v1.15.4'
+```
+
+#### Helm init
 
-Using the command line, run first this command:
+The first time you install Substra, you will need to use:
 
 ```sh
 helm init
+# or
+helm init --upgrade
+# you might need to use
+helm init --service-account tiller --upgrade
 ```
 
+#### Network
+
+- Ubuntu
+
+```sh
+# Append your Kubernetes cluster ip to your system hosts
+echo "$(minikube ip) substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com" | sudo tee -a /etc/hosts
+
+# Inside a VM, you will need to use
+echo "$(sudo minikube ip) substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com" | sudo tee -a /etc/hosts
+```
+
+Example:
+
+```sh
+192.168.39.32 substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com
+```
+
+If you want to customize your configuration,
+you can assign the ingress loadbalancer ip to the corresponding url, for example:
+
+```sh
+10.111.206.49 substra-backend.node-1.com substra-frontend.node-1.com
+10.109.77.31 substra-backend.node-2.com substra-frontend.node-2.com
+```
+
+If you want to expose another service, you can use something like:
+
+```sh
+kubectl port-forward -n ${NAMESPACE} service/${SERVICE_NAME} ${SERVICE_PORT}:${PORT_WANTED}
+```
+
+- Mac
+
 Once done, we can configure your machine so that the urls `substra-backend.node-1.com`, `substra-backend.node-2.com`, `substra-frontend.node-1.com` and `substra-frontend.node-2.com` point to your local instance of the platform.
```

Comment:
It seems to be valid for both Ubuntu and Mac. Maybe it could be moved at the top of Network part?

Reply:
Bien vu, merci @samlesu

Reply:
Sorry for the french moment, thanks for your input @samlesu